src/OpenIddict.Client.WebIntegration/OpenIddictClientWebIntegrationHandlers.cs

```diff
@@ -1585,9 +1591,12 @@ ProviderTypes.Webflow       or ProviderTypes.Weibo      or ProviderTypes.Yandex
                 // WordPress returns the user identifier as a custom "ID" node:
                 ProviderTypes.WordPress => (string?) context.UserInfoResponse?["ID"],
 
-                // WordPress returns the user identifier as a custom "ZUID" node:
+                // Zoho returns the user identifier as a custom "ZUID" node:
                 ProviderTypes.Zoho => (string?) context.UserInfoResponse?["ZUID"],
 
+                // Genesys returns the user identifier as a custom "ZUID" node:
+                ProviderTypes.Genesys => context.MergedPrincipal.GetClaim("id"),
```

Comment:
Assuming the user identifier is returned as an `id` userinfo node, remove that and add the provider type to the switch case line 1522.